_infra/setup-dev-station/install.os.tool.yq.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Habits
```

Comment:
Lines 2, 3 and 4 look wierd like they're from example. Are they OK?

Reply:
fixed